docker_start.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+PORT=3000
+
+export NODE_PORT=${PORT}
+exec node /opt/server.js -- ${PORT}
```

Comment:
I have a doubt about this line.
Is it not rather:
node opt/dist/bin/www.js

Reply:
i think the COPy command in Dockerfile will copy the contents of dist, so you won't need dist in the path

Reply:
edited line